torch/nn/modules/activation.py

```diff
@@ -299,6 +300,43 @@ def extra_repr(self):
         return 'alpha={}{}'.format(self.alpha, inplace_str)
 
 
+class CELU(Module):
+    r"""Applies element-wise,
+    :math:`\text{CELU}(x) = \max(0,x) + \min(0, \alpha * (\exp(x/\alpha) - 1))`
+
+    More details can be found in the paper `Continuously Differentiable Exponential Linear Units`_ .
```

Comment:
This paper is missing a link. Here is an example on how to add it: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/batchnorm.py#L151-L152

Reply:
fixed